paasJobs/scripts/gc_crawl_then_upload.sh

```diff
@@ -83,8 +88,28 @@ function recreate_host_dl_dir() {
 }
 
 function grab_manifest() {
-  echo -e "\nGRABBING LATEST MANIFEST\n"
-  (aws s3 cp ${S3FULLPATH_MANIFEST} ${LOCAL_PREVIOUS_MANIFEST_LOCATION}) || echo -e "\nFAILED TO GRAB MANIFEST\n"
+  local rc
+  >&2 echo -e "\n[INFO] GRABBING LATEST MANIFEST\n"
+  aws s3 cp "${S3FULLPATH_MANIFEST}" "${LOCAL_PREVIOUS_MANIFEST_LOCATION}" && rc=$? || rc=$?
+
+  if [[ "$rc" -ne 0 ]]; then
+    >&2 echo -e "\n[ERROR] FAILED TO GRAB MANIFEST\n"
+    exit 11
+  fi
+}
+
+function update_manifest() {
+  local inner_job_dl_dir="${HOST_JOB_DL_DIR}/$(basename "$CRAWLER_CONTAINER_DL_DIR")"
+  local local_new_cumulative_manifest="${inner_job_dl_dir}/cumulative-manifest.json"
+  local s3_backup_cumulative_manifest="${S3FULLPATH_MANIFEST%.json}.${JOB_TS_SIMPLE}.json"
+
+  >&2 echo -e "\n[INFO] UPDATING LATEST MANIFEST\n"
+  # backup old manifest
+  aws s3 cp "${S3FULLPATH_MANIFEST}" "$s3_backup_cumulative_manifest" \
```

Comment:
really minor, just put the curly braces around s3_backup_cumulative_manifest for consistency. otherwise all looks good